maven-model-builder/src/test/java/org/apache/maven/model/validation/DefaultModelValidatorTest.java

```diff
@@ -607,15 +607,12 @@ public void testBadImportScopeType()
                         "'dependencyManagement.dependencies.dependency.type' for test:a:jar must be 'pom'" );
     }
 
-    public void testBadImportScopeClassifier()
+    public void testImportScopeClassifier()
         throws Exception
     {
-        SimpleProblemCollector result = validateRaw( "bad-import-scope-classifier.xml" );
+        SimpleProblemCollector result = validateRaw( "import-scope-classifier.xml" );
 
-        assertViolations( result, 0, 1, 0 );
-
-        assertContains( result.getErrors().get( 0 ),
-                        "'dependencyManagement.dependencies.dependency.classifier' for test:a:pom:cls must be empty" );
+        assertViolations( result, 0, 0, 0 );
```

Comment:
If this is meant to signify that a violation is not thrown, would it be possible to create a method assertNoViolations( result ) ?  Personally I feel as though it parses better.

Reply:
Hi @KathrynN, thank you for looking at the diff!

The `assertViolations( result, 0, 0, 0 );` is used in 5 other test methods in addition to the one I've edited. Changing these would expand the scope of this concise diff to those unrelated test cases - is it, in your opinion, justified?

Reply:
Possibly in another story/issue, not here!  Maybe mark that refactoring as good for beginners too :)
Thanks for the swift response